### PR TITLE
#12 ready for review

### PR DESCRIPTION
**Pull request by Lajmeno**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/app/Main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package de.neuefische.app;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Main {
```
Comment (gossie) [inline backend/src/main/java/de/neuefische/app/Main.java]: Why was the class commited?

File: backend/src/main/java/de/neuefische/app/playlist/PlaylistDTO.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class PlaylistDTO {
+
+    private String name;
+    private String spotifyId;
+    private List<PlaylistTrack> tracks;
```
Comment (gossie) [inline backend/src/main/java/de/neuefische/app/playlist/PlaylistDTO.java]: PlaylistTrack and PlaylistImage are the classes that you also use in the PlaylistData class. If you seperate data and DTO you should do it all the way.

File: backend/src/main/java/de/neuefische/app/playlist/PlaylistService.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class PlaylistService {
+
+    private final PlaylistRepository playlistRepository;
+
+    public Optional<PlaylistDTO> savePlaylist(PlaylistData playlistData){
```
Comment (gossie) [inline backend/src/main/java/de/neuefische/app/playlist/PlaylistService.java]: In my opinion the Service should not know about the DTO. DTOs are only relevant in the controller

**Review by gossie — CHANGES_REQUESTED** [review]

File: backend/src/main/java/de/neuefische/app/Main.java (hunk 1 above)
Comment (Lajmeno) [reply backend/src/main/java/de/neuefische/app/Main.java]: Yes I should just delete it, I used it before to check somethin

File: backend/src/main/java/de/neuefische/app/playlist/PlaylistDTO.java (hunk 2 above)
Comment (Lajmeno) [reply backend/src/main/java/de/neuefische/app/playlist/PlaylistDTO.java]: Ok, I am going to create a PlaylistTrackDTO and ImageDTO

File: backend/src/main/java/de/neuefische/app/playlist/PlaylistService.java (hunk 3 above)
Comment (Lajmeno) [reply backend/src/main/java/de/neuefische/app/playlist/PlaylistService.java]: Good to know! I will change that in the following commits, when I have a PlaylistController

**Review by gossie — APPROVED** [review]

**Pull request merged**